addon/components/cfb-form-editor/general.js

```diff
@@ -27,20 +28,25 @@ export default Component.extend(ComponentQueryManager, {
 
   data: task(function*() {
     if (!this.get("slug")) {
-      return {
-        node: {
-          name: "",
-          slug: "",
-          description: ""
+      return A([
+        {
+          node: {
+            name: "",
+            slug: "",
+            description: ""
+          }
         }
-      };
+      ]);
```

Comment:
Why do you need the ember array?

Reply:
Because we now use the `allForms` graph instead of `node`. Node would return an object like `{ node: { title: "Form #1" } }` whereas allForms returns something like `{ allForms: { edges: [ { node: { title: "Form #1" } } ] } }`. In the template we're using `result.firstObject.node` then..